### ISSUE TITLE
Press overview

### ISSUE THREAD

**Issue by Baschdl** [Baschdl/match4healthcare#535]
Would should be included? Name of publisher, name of publication, date, link?

Can we include a logo?

@maltezacharias

**Comment by maltezacharias** [comment]
I'm unable/unsure if there was already a list. The basic layout should include:
- a heading
- an image (e.g. Logo) 
- a short excerpt (written by us) 
- a read more link 
- a publishing date

We discussed something like this: https://www.amnesty.at/presse/ I am not a huge fan of that though. I would prefer something more like this:
![image](https://user-images.githubusercontent.com/9914287/85181643-b88f9200-b286-11ea-9ecf-3be70196814a.png)

I think the alternating left-right-layout is beautiful. We should evaluate in m4e if we can do this in wagtail, I think this would be the proper way of doing it @kevihiiin 
@andreaskind do we have a list of press mentions compiled already?

**Comment by maltezacharias** [comment]
This is also an example more like what the team proposed:
https://hootsuite.com/resources?tags%5B%5D=Case+Study

**Comment by Baschdl** [comment]
I also though about the logo but this could be problematic regarding copyright

**Comment by Baschdl** [comment]
Wagtail or the right CMS is a big open question: https://github.com/match4everyone/match4everything/issues/25

**Comment by maltezacharias** [comment]
> I also though about the logo but this could be problematic regarding copyright

Good point. But we do have the permission from some of the sites, so we should be able to use this for at least some entries

**Comment by Baschdl** [comment]
I'm no UI expert but I always find it strange if there are only images for some entries and placeholders for the others

**Comment by maltezacharias** [comment]
I'll ask the whatsapp Team to acquire permissions :)
